### ISSUE TITLE
Format incorrect when columnLimit is set to 0

### ISSUE THREAD

**Issue by EternalPhane** [mike-lischke/vscode-antlr4#11]
When setting `antlr4.format.columnLimit` to `0`, the format becomes weird:

![image](https://user-images.githubusercontent.com/11267872/33524443-7728c0b2-d858-11e7-9219-95949867d2b1.png)

comparing to what I expect:

![image](https://user-images.githubusercontent.com/11267872/33524464-bf7b2df0-d858-11e7-853b-dfa31e6a9467.png)

My configuration is like below:
```js
{
    "antlr4.format.alignColons": "hanging",
    "antlr4.format.alignSemicolons": "ownLine",
    "antlr4.format.allowShortRulesOnASingleLine": false,
    //"antlr4.format.columnLimit": 0,
    "antlr4.format.columnLimit": 10000,
    "antlr4.format.useTab": false,
    "antlr4.generation.mode": "external",
    "antlr4.generation.outputDir": "/.antlr",
}
```

**Comment by mike-lischke** [comment]
A value of 0 should ignore the column limit, right.

**Comment by mike-lischke** [comment]
Fixed by b5f1737.
